src/key_value/key_value.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021, USC
+// All rights reserved.
+
+#ifndef SRC_KEY_VALUE_KEY_VALUE_H_
+#define SRC_KEY_VALUE_KEY_VALUE_H_
+
+#include <mutex>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace csci499 {
+
+// key value storage for csci499 application
+class KeyValue {
+ public:
+  KeyValue() : storage_(), lock_() {}
+
+  virtual ~KeyValue() {}
+
+  void Put(const std::string& key, const std::string& value);
+  std::vector<std::string> Get(const std::string& key);
```

Comment:
Could make Get() a const method.

Reply:
Would this work even if I have to call lock on the mutex lock?